source/PlanetPanel.cpp

```diff
@@ -367,12 +368,19 @@ void PlanetPanel::CheckWarningsAndTakeOff()
 			out << " that will not be able to leave the system.";
 		}
 		// Warn about non-commodity cargo you will have to sell.
-		else
+		else if(hasSpaceport)
```

Comment:
I think we can have planets with an outfitter, but without a spaceport, so this should probably check for CanUseServices && (HasSpaceport || HasOutfitter).

Reply:
I can't check rn but you could also delete the whole if statement that checks for outfit costs I think.